### PR TITLE
fix a few minor code smells

### PR DESCRIPTION
**Pull request by LukasReschke**
From https://github.com/owncloud/core/pull/25828

### CODE REVIEW

File: apps/user_ldap/lib/Connection.php [hunk 1]
```diff
@@ -580,6 +581,7 @@ private function doConnect($host, $port) {
 		} else {
 			throw new \OC\ServerNotAvailableException('Could not set required LDAP Protocol version.');
 		}
+		return true;
```
Comment (nickvergessen) [inline apps/user_ldap/lib/Connection.php]: True is not listed as return value in the doc block

Comment (rullzer) [inline apps/user_ldap/lib/Connection.php]: Mmmm as I understand returning should be types and not values. So yeah should be modified to return bool

Comment (rullzer) [inline apps/user_ldap/lib/Connection.php]: fix the docs.

**Comment by rullzer** [comment]
:+1: now from me

**Comment by nickvergessen** [comment]
:+1:

**Pull request merged**